python/create_mask.py

```diff
@@ -0,0 +1,138 @@
+"""
+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
+;
+; Name        : create_mask
+;
+; Purpose     : Create transparent CH masks from previously stored data
+;
+; Syntax      : create_mask.py path 
+;
+; Inputs      : CHIMERA ch_location.txt file
+;
+; Outputs     : Transparent CH mask
+;
+; Keywords    : path = directory path of file	
+;
+; Example     : ipython> % run -i create_mask.py '/path/to/file/location/'
+;
+; Required Packages    : pip install mahotas
+;
+; History     : v1.0, Written 14-Jun-2018, Tadhg Garton, TCD
+;
+; Contact     : gartont@tcd.ie
+;		info@solarmonitor.org
+
+"""
+
+from datetime import datetime
+import glob
+import sys
+
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+import os.path as op
+
+#=====Check python version=====
+
+if sys.version[0] != '3':
+    raise NameError('=====   Python version 3 must be used for create_mask.py   =====')
+
+#=====Establish pathing=====
+path = ''
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+    exis = op.exists(op.join(path,'meta/'))
+    if exis == 0:
+        raise NameError('===== Path to CH location file does not exist =====')
+
+#=====Find CH location file=====
+f = glob.glob(op.join(path,"meta/*ch_location*.txt"))
+
+#=====Establish variables and arrays=====
+siz = 4096
+hsiz=siz/2
+iarr = np.zeros((siz, siz) , dtype=np.byte)
+slate, circ = np.array(iarr), np.array(iarr)
+x, y = np.array([]), np.array([])
+fill = 0
+
+#=====Cycle through lines in file=====
+with open(f[0],"r") as file:
```

Comment:
Have you looked to [`astropy.io.ascii`] (http://docs.astropy.org/en/stable/io/ascii/) `? Maybe there's a reader that you can use. If not I'll suggest to write the reader as a separate function that you can test independently and reuse in other cases.